docs/pages/eas-update/standalone-service.mdx

```diff
@@ -21,6 +22,28 @@ That said, many organizations are already heavily invested in their CI/CD infras
 
 ## Using EAS Update without EAS Build
 
-Most of the [installation and configuration steps](/eas-update/getting-started) are identical whether you are using EAS Build or not. The primary difference is that you will need to configure the update [channel](/eas-update/eas-cli/) manually.
-
-A build of your app points to a single EAS Update channel that it will pull updates from, such as `preview` or `production`. Before you create a build, you'll need to configure the channel name. If you create a build with EAS Build, the channel name from **eas.json** will automatically be added to our build's **AndroidManifest.xml** and **Expo.plist** at build time. The primary difference between using EAS Update without EAS Build is that you will need to configure this field manually.
+Most of the [installation and configuration steps](/eas-update/getting-started) are identical whether or not you use EAS Build. The primary difference is how the update [channel](/eas-update/eas-cli/) is configured. When using EAS Build, the channel from **eas.json** will automatically be added to your build's **AndroidManifest.xml** and **Expo.plist** at build time. When not using EAS build, this must be configured manually by setting the request header in the app config, as shown below, followed by manually creating the channel on the server.
+
+{/* prettier-ignore */}
+```json app.json
+  "expo": {
+    /* @hide ... */ /* @end */
+    "updates": {
+      /* @hide ... */ /* @end */
+      "requestHeaders": {
+        "expo-channel-name": "main"
+      }
+      /* @hide ... */ /* @end */
+    }
+    "runtimeVersion": "1.0.0",
+    /* @hide ... */ /* @end */
+  }
+```
```

Comment:
this isn't necessary, it's here already: https://docs.expo.dev/eas-update/getting-started/#configure-update-channels-in-appjson

Reply:
Good call! I'll update to link to that.